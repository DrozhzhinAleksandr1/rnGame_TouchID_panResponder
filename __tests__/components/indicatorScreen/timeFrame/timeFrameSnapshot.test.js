import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { TimeFrame } from '../../../../app/components/indicatorScreen/timeFrame/timeFrame';

it('create snapshot TimeFrame', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<TimeFrame/>);
    expect(toolbarSnapshot).toMatchSnapshot();
});
