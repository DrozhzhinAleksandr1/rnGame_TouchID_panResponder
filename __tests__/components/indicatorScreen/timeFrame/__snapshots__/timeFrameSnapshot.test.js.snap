// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot TimeFrame 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#172533",
      "height": 40,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Component
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <FrameBtn
      frameBtnText="1 мин"
    />
    <FrameBtn
      frameBtnText="5 мин"
    />
    <FrameBtn
      frameBtnText="15 мин"
    />
    <FrameBtn
      frameBtnText="30 мин"
    />
    <FrameBtn
      frameBtnText="1 час"
    />
  </Component>
</Component>
`;
